### DISCUSSION TITLE
Tree组件可以实现点击文字展开或折叠吗？没有找到相关属性

### DISCUSSION THREAD

**Discussion by whale-jung** [ant-design/ant-design#43906] [Q&A]
Tree组件可以实现点击文字展开或折叠吗？没有找到相关属性

**Comment by afc163** [comment]
https://ant.design/components/tree-cn#components-tree-demo-directory

**Reply by whale-jung** [reply]
样式可以配置成左边的样式吗

<img width="698" alt="image" src="https://github.com/ant-design/ant-design/assets/80868414/9fee1bc1-6b6b-4aac-bebe-e6101809be9b">
